spec.md

```diff
@@ -377,24 +377,31 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
```

Comment:
This name is confusing, because it's in plural, but it actually denotes a single object containing multiple schemas. It should be renamed to _JSON Schemas Object_ or something along the lines of _Input/Output Schema Object_.

Reply:
The rational was because the json schema may contain multiple object definitions... But yeah calling it just `Json Schema Object` is better.